menas/src/main/scala/za/co/absa/enceladus/menas/services/PropertyDefinitionService.scala

```diff
@@ -34,7 +34,11 @@ class PropertyDefinitionService @Autowired()(propertyDefMongoRepository: Propert
 
   override def update(username: String, propertyDef: PropertyDefinition): Future[Option[PropertyDefinition]] = {
     super.update(username, propertyDef.name, propertyDef.version) { latest =>
-      latest.setDescription(propertyDef.description)
+      latest
```

Comment:
Why using the setters here, creating 4 new instances, instead of using the copy constructor directly, once, with the four properties?

Reply:
Keeping up with what is already established in dataset, schema and mapping table services

Reply:
Not blocking for sure. Just considered it little ineffective. 😉